### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by MICHAELMUNAVU83**
:wave: Hello reviewer. Thanks for taking out time to view this PR. Here's a rundown of the milestones completed for this project.

:heavy_check_mark: Display a splash screen for new users visiting the app. It leads to a sign-in and login page
:heavy_check_mark:Users can sign up with their name, email, and password
:heavy_check_mark:Users can create a category and upload an icon for it
:heavy_check_mark:Users can create payments/transactions for each category
:heavy_check_mark:The app is styled according to the template provided
:heavy_check_mark:Unit and integrations have been written and asserted to confirm the behavior of the app matches the project expectations
:link: Live link available [here](https://morning-chamber-16356.herokuapp.com/)
:movie_camera: Video presentation available [here](https://www.loom.com/share/792c0285d6574a57bff77b2eb16768ba)

### CODE REVIEW

File: app/assets/config/manifest.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
 //= link_tree ../images
-//= link_directory ../stylesheets .css
 //= link_tree ../../javascript .js
 //= link_tree ../../../vendor/javascript .js
+//= link_tree ../builds
```
Comment (OLIPLICHE) [inline app/assets/config/manifest.js]: - [ ] Please provide a sign-up and log-in pages

- [ ] Also, you have more than one unused(empty) files in Assets folder you implemented. you can remove them to make your work clean 

Capture

File: app/models/user.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
+  has_many :categories, dependent: :destroy
+  has_many :payments, dependent: :destroy
+  validates :name, presence: true, length: { minimum: 3, maximum: 25 }
+
+  devise :database_authenticatable, :registerable, :recoverable, :rememberable, :validatable
```
Comment (OLIPLICHE) [inline app/models/user.rb:6]: - Nice job in following the Techincal requirements 👍 👍 🥇

File: spec/features/categories_spec.rb [hunk 3]
```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
```
Comment (OLIPLICHE) [inline spec/features/categories_spec.rb:1]: - Please, This is what I am getting: "rspec spec"
- Make sure that all the test pass.

Capture

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @MICHAELMUNAVU83, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Done a Nice video presentation👏 👏
- Linter is green (the last commit have a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏
- Githflow is correctly used (Pull Request is made from the feature branch into the dev branch) 👏

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by MICHAELMUNAVU83** [comment]
Hello next reviewer , kindly run ` rails db:migrate RAILS_ENV=test` to migrate to test environment as they are passing
Screenshot from 2022-10-18 11-57-20
The login in and sign up pages are available
Screenshot from 2022-10-18 12-03-15
- You can view the site here to see
 :link: Live link available [here](https://morning-chamber-16356.herokuapp.com/)

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @MICHAELMUNAVU83,

✔️ Passing tests

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**